src/components/anchor-list-item/anchor-list-item.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../styles/global-css-settings";
+@import "../../styles/typography";
+
+.monday-storybook-anchor-list-item {
```

Comment:
[Optional]

If it not super difficult to change prefix for classNames - I would change it to something like `vibe-storybook...` or `vibe-sb-...`